README.md

```diff
@@ -146,7 +195,11 @@ See: [results.md](results.md)
 * [ ] worth noting the difference between f32->f64->f128 and integer rational
   i32->i64 etc.
 * [ ] Fold other references from [^1] into the document.
+* [ ] Verify the sampling hypothesis that numbers near the limit may be
+  reversibly brought near the origin and put back again
```

Comment:
Yeah.  Would love to see an experiment of this.  @meshula would you want that to be in a follow up PR?

Reply:
yes, a follow up.